### DISCUSSION TITLE
Is it possible to run single server components on its own?

### DISCUSSION THREAD

**Discussion by Rindula** [DarkflameUniverse/DarkflameServer#1589] [Q&A] [answered]
The intention is, to run the master server somwhere, the authserver somwhere else, chatserver also and especially the world servers on their own.

This would enable me to scale the setup inside a kubernetes cluster, like world servers on demand and saving resources if not needed anymore. This would also require some sort of metrics endpoint (which I currently emulate by running ncat with a basic prometheus response style in a while loop) or some sort of player count readable on the worldservers

**Comment by aronwk-aaron** [comment, marked as answer]
Yes, @Xiphoseer is working on K8 support and had a POC PR up. Just limited time to finish it currently

**Reply by Rindula** [reply]
Nice, found it, thanks a lot!

For reference, this should be PR #1378
